etc/grafana/dashboards/reth-mempool.json

```diff
@@ -2679,6 +2679,176 @@
       "title": "Fetch Hashes Pending Fetch Duration",
       "type": "timeseries"
     },
+    {
+      "datasource": {
+        "type": "prometheus",
+        "uid": "PBFA97CFB590B2093"
```

Comment:
you need to choose export for public use in grafana
<img width="677" alt="Screenshot 2024-10-07 at 13 47 16" src="https://github.com/user-attachments/assets/ce82b639-e5ad-4236-9a14-026e940a4a8d">
click the share button in top right corner, then export in the menu, to get here